src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const createMessage = (text, type) => {
+  const newMessage = document.createElement('div');
+
+  newMessage.dataset.qa = 'notification';
+
+  newMessage.className = `message ${type}`;
+
+  newMessage.textContent = text;
+
+  document.body.append(newMessage);
+};
+
+const firstPromise = new Promise((resolve, reject) => {
+  let click = false;
+
+  document.addEventListener('click', e => {
+    click = true;
+
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    if (click === false) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('contextmenu', e => {
+    resolve('Second promise was resolved');
+  });
+});
```

Comment:
Pay attention to this requirement
![image](https://user-images.githubusercontent.com/98764641/222359983-e5ca1984-4d01-4f8f-bf99-7d0c2e1b7374.png)